app/features/ledger/Transfer.ts

```diff
@@ -57,6 +60,38 @@ const INS_REMOVE_BAKER = 0x14;
 const INS_UPDATE_BAKER_STAKE = 0x15;
 const INS_UPDATE_BAKER_RESTAKE_EARNINGS = 0x16;
 
+type Memo = string;
+
+/**
+ * Send memo to ledger for signing.
+ * N.B. This does not send the length of the memo, and the CBOR encoded string.
+ */
+async function sendMemo(
+    transport: Transport,
+    ins: number,
+    memo: Memo | undefined,
+    p1: number,
+    p2: number
+) {
+    if (!memo) {
+        throw new Error('Unexpected Missing memo');
+    }
+
+    const encodedMemo = encodeAsCBOR(memo);
+
+    let response;
+    const chunks = chunkBuffer(encodedMemo, 255);
+    for (const chunk of chunks) {
+        // eslint-disable-next-line  no-await-in-loop
+        response = await transport.send(0xe0, ins, p1, p2, Buffer.from(chunk));
+    }
+    if (!response) {
+        throw new Error('Unexpected missing response from ledger;');
+    }
+    const signature = response.slice(0, 64);
```

Comment:
Else you refactored this from
```
    const signature = response.slice(0, 64);
    return signature;
```
to
```
    return response.slice(0, 64);
```
Pick one and stay consistent.

Reply:
I have refactored it for all Transfers.